### PR TITLE
Add avatar in og-image

### PR DESCRIPTION
**Pull request by nachoiacovino**

### CODE REVIEW

**Review by jnsdls — APPROVED** [review]
LGTM

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**…ontractName\]/\[version\]**](https://thirdweb-m4glrwxuv-thirdweb.vercel.app/contracts/[wallet]/[contractName]/[version]) |
|:-:|
|<a href="https://thirdweb-m4glrwxuv-thirdweb.vercel.app/contracts/[wallet]/[contractName]/[version]">Screenshot of /contracts/[wallet]/[contractName]/[version]</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/N89hqvvp-biCu23MFBxJ8ELPeHGN8uaPUuyEHDZWLZsGKJXLJZb6-pWwFwvmYRULdgQ7DFb2_wtaa51EIOkB-g.png">(view full size)</a> |







Commit bae3dfa45abc64665609746ac4dd7938b402eb3e (https://thirdweb-m4glrwxuv-thirdweb.vercel.app).

File: components/contract-components/hooks.ts [hunk 1]
```diff
@@ -223,7 +221,7 @@ export function useAllVersions(
 ) {
   const sdk = useSDK();
   return useQuery(
-    ["all-releases", publisherAddress, contractName],
+    ["all-releases-2", publisherAddress, contractName],
```
Comment (jnsdls) [inline components/contract-components/hooks.ts]: this is real sketch, why?

File: components/contract-components/released-contract/index.tsx [hunk 2]
```diff
@@ -106,9 +108,17 @@ export const ReleasedContract: React.FC<ReleasedContractProps> = ({
         });
     }
     url.searchParams.append("releaser", releaserEnsOrAddress);
-    url.searchParams.append("avatar", release.avatar);
+    url.searchParams.append("avatar", releaserProfile.data?.avatar || "");
```
Comment (jnsdls) [inline components/contract-components/released-contract/index.tsx:111]: just don't append it if we don't have it?

**Review by jnsdls — CHANGES_REQUESTED** [review]
some quick things to look at

File: components/contract-components/hooks.ts (hunk 1 above)
Comment (nachoiacovino) [reply components/contract-components/hooks.ts]: I was testing if it was a cache issue, already reverted by the time you commented, I was so confused :D

**Review by jnsdls — APPROVED** [review]
LGTM

**Pull request merged**